### DISCUSSION TITLE
Как добавить устройство в конфиг???

### DISCUSSION THREAD

**Discussion by ogizesh** [Mon4ik/homebridge-yandex#7] [General]
Как добавить устройство в конфиг???

**Comment by Mon4ik** [comment]
Он должен сам подтягиваться с умного дома, надо только клиент настроить через oAuth

**Reply by ogizesh** [reply]
А что в клиенте нужно делать? Там вообще настроек никаких

**Reply by ogizesh** [reply]
Нашёл доступы к iot, но устройств не вижу. Как он понимает, какие устройства нужно брать? Из аккаунта?
